std/experimental/ndslice/slice.d

```diff
@@ -1178,6 +1178,29 @@ struct Slice(size_t _N, _Range)
             _ptr._range = range;
     }
 
+    /++
+    Returns:
+        pointer to the first element of a slice if slice is defined as `Slice!(N, T*)` or
+        plain structure with two fields `shift` and `range` otherwise. In second case the expression `range[shift]`
+        refers to the first element. For slices with named elements the type of a return value has the same behavior like a pointer.
+    Note:
+        `ptr` is defined only for not packed slices.
```

Comment:
Grammatically, this should be "*non-packed slices*" or "*slices that are not packed*".